source/ore/index.rst

```diff
@@ -0,0 +1,21 @@
+=================
+Ore Documentation
+=================
+
+Our custom built plugin hosting solution provides a unique and convenient way for content creators to share and
+make accessible plugins that they have created. For developers, Ore offers a centralized location where their users
+can download, review, and get support on their plugin. For server administrators, Ore offers a level of safety and
+security previously lacking from many Minecraft communities.
```

Comment:
Subtle dig. Perhaps stating how we accomplish "safety and security" would be more informative.  ie. a staff-reviewed(?) secure repository.